src/tests/build.sh

```diff
@@ -318,6 +320,30 @@ handle_arguments_local() {
             fi
             ;;
 
+        target-os*|-target-os*)
```

Comment:
I think it would be better to structure this the same as the Windows and `run.sh` script: Add `wasi`/`browser` arguments (possibly also `wasm`, though it's not strictly required). It would make the command lines more consistent and simplify CI.

Reply:
thanks, done.